services/director-v2/tests/integration/02/test_mixed_dynamic_sidecar_and_legacy_project.py

```diff
@@ -220,13 +215,13 @@ async def test_legacy_and_dynamic_sidecar_run(
     - dy-static-file-server-dynamic-sidecar
     - dy-static-file-server-dynamic-sidecar-compose
     """
-    director_v0_url = get_director_v0_patched_url(services_endpoint["director"])
+    # FIXME: ANE can you instead parametrize this test?
+    # why do we need to run all these services at the same time? it would be simpler one by one
```

Comment:
The idea behind this was to test that these services could all run in parallel. Why would you change it to only one service?

Reply:
to simplify debugging